src/network/networkd-ndisc.c

```diff
@@ -368,10 +371,19 @@ static int ndisc_router_process_autonomous_prefix(Link *link, sd_ndisc_router *r
         if (r < 0)
                 return log_link_error_errno(link, r, "Failed to get prefix valid lifetime: %m");
 
+        r = sd_ndisc_router_get_lifetime(rt, &router_lifetime);
+        if (r < 0)
```

Comment:
Does this value always appears or can be ignored ? If it's not mandatory then we should not return here at all.  Cap should be calculated only if this value is present or the code the the same as it is.

Reply:
DOnes as suggested.